File: Two-Sum/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        //sort(nums.begin(), nums.end());
        int l=nums.size();
        vector<int> ans(2);
        int i=0;
        while(i<l-1){
            int x=nums[i];
            for(int j=i+1;j<l;j++){
                int y=nums[j];
                if(x+y == target){
                    ans[0]=i; ans[1]=j;
                    return ans;
                }
            }
            i++;
        }
        return ans;
    }
};

File: Two-Sum-II/two-sum.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int l = numbers.length;
        int i=0;
        int j=l-1;
        int sum=numbers[i]+numbers[j];
        while(sum!=target){
            if(sum>target)
                j--;
            else
                i++;
            sum = numbers[i]+numbers[j];
        }
        int[] lis={i+1, j+1};
        return lis;
    }
}
